Unix/configeditor/configeditor.cpp

```diff
@@ -372,7 +392,10 @@ int MI_MAIN_CALL main(int argc, const char** argv)
         // If property isn't the one we want, move along
         if (prop != arg1)
         {
-            cout << line << std::endl;
+            if (!fDontEchoFile)
```

Comment:
If you don't want to output the omiserver.conf file, that's fine.

Outputting the string "has value" or "does not have value" means that callers, at least from postinstall, will need to redirect stdout to /dev/null. That's fine, and is consistent with other Linux utilities (grep, etc). Just pointing this out so you're aware when you start using this new feature.

On combining the `if` statements, brain fart on my part. I was looking/reviewing too quickly. Of course you're right, you can't combine in this case.